modules/bootstrap-aws-load-balancer-operator.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+// * networking/installing-albo-sts-cluster.adoc
+
+:_content-type: PROCEDURE
+[id="nw-bootstra-albo-on-sts-cluster_{context}"]
+= Bootstrapping AWS Load Balancer Operator on Secure Token Service cluster
+
+.Prerequisites
+
+* You must extract and prepare the `coctl` binary.
+
+.Procedure
+
+. Use the `ccoctl` tool to process the operator's `CredentialsRequest` objects needed to bootstrap the operator, by running the following command:
++
+[source,terminal]
+----
+$ curl --create-dirs -o <path-to-credrequests-dir>/cr.yaml https://raw.githubusercontent.com/openshift/aws-load-balancer-operator/main/hack/operator-credentials-request.yaml
+ccoctl aws create-iam-roles \
+    --name <name> --region=<aws_region> \
+    --credentials-requests-dir=<path-to-credrequests-dir> \
+    --identity-provider-arn <oidc-arn>
+----
+
+For each `CredentialsRequest` object, `ccoctl` creates an IAM role with a trust policy that is tied to the specified OIDC identity provider, and permissions policy as defined in each `CredentialsRequest` object. This also generates a set of secrets in a manifests directory that is required by the AWS Load Balancer Operator.
+
+. Apply the secrets to your cluster, by running the following command:
++
+[source,terminal]
+----
+$ ls manifests/*-credentials.yaml | xargs -I{} oc apply -f {}
+----
+
+. Verify that the credentials secret of the AWS Load Balancer Operator is created, by running the following command:
++
+[source,terminal]
+----
+$ oc -n aws-load-balancer-operator get secret aws-load-balancer-operator -o json | jq -r '.data.credentials' | base64 -d
```

Comment:
```suggestion
$ oc -n aws-load-balancer-operator get secret aws-load-balancer-operator --template='{{index .data "credentials"}}' | base64 -d
```
Just recalled that `jq` utility is better to be avoided.

Reply:
acked